packages/graphback-runtime-knex/src/knexQueryMapper.ts

```diff
@@ -0,0 +1,108 @@
+import Knex from 'knex';
+
+interface OperatorMap {
+  ne: '<>',
+  eq: '=',
+  le: '<=',
+  lt: '<',
+  ge: '>=',
+  gt: '>',
+  contains: 'like',
+  startswith: 'like',
+  endswith: 'like',
+}
+
+const methodMapping = {
+  in: 'whereIn',
+  nin: 'whereNotIn',
+  between: 'whereBetween',
+  nbetween: 'whereNotBetween',
+  default: 'where'
+}
+
+const AND_FIELD = 'and';
+const OR_FIELD = 'or';
+const NOT_FIELD = 'not';
+
+function mapOperator(operator: any) {
+  const operatorMap: OperatorMap = {
+    ne: '<>',
+    eq: '=',
+    le: '<=',
+    lt: '<',
+    ge: '>=',
+    gt: '>',
+    contains: 'like',
+    startswith: 'like',
+    endswith: 'like'
+  }
+
+  const oprLower = operator.toLowerCase()
+
+  if (!Object.keys(operatorMap).includes(oprLower)) {
+    throw Error(`Not supported operator: ${operator}`)
+  }
+
+  return operatorMap[oprLower]
+}
+
+function builderMethod(method: string, clause?: string) {
+  switch (clause) {
+    case 'not':
+      return `${method}Not`;
+    case 'or':
+      return `${clause}${method.charAt(0).toUpperCase()}${method.slice(1)}`;
+    default:
+      return method;
+  }
+}
+
+function where(builder: Knex.QueryBuilder, filter: any, clause?: string) {
```

Comment:
Similar to graphql-server-crud with some changes

https://github.com/charlie0077/graphql-server-crud/blob/35c04ab06c6dd05d2c4cc35b50a2d06aeacada81/src/knexTransform.js#L61